order.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="src/index.css">
+    <link rel="stylesheet" href="src/order.css">
+</head>
+<body>
+    <header>
+        <h1>Order Details</h1>
+    </header>
+    <main>
+        <dl>
+            <dt>Name</dt>
+            <dd id="name"></dd>
+        
+            <dt>Phone Number</dt>
+            <dd id="phone-number"></dd>
+
+            <dt>Return Customer</dt>
+            <dd id="return-customer"></dd>
+
+            <dt>Item Number</dt>
+            <dd id="item-number"></dd>
+
+            <dt>Item Description</dt>
+            <dd id="item-description"></dd>
+
+            <dt>Price</dt>
+            <dd id="price"></dd>
+        </dl>
+    </main>
+    
+    <script type="module" src="src/order.js"></script>
+</body>
+</html>
+    
+    
+    
+    
```

Comment:
Same as before. 1 or 2 lines is fine, but anymore is just clutter. Its fine when you are actually coding, but when you go to turn it in or push to production in the real world, having too much white space is just clutter.